src/sentry/static/sentry/app/views/organizationDiscover/styles.jsx

```diff
@@ -50,6 +50,11 @@ export const BodyContent = styled(Flex)`
   padding: ${space(1.5)} 32px 32px 32px;
 `;
 
+export const Loading = styled(Flex)`
```

Comment:
Loading is a bit ambiguous since what you are importing does not include anything involving loading. maybe `BottomBarAwareContainer`?

Or if you don't think this code will be used again, you could just import `FOOTER_HEIGHT` in `index.jsx` and write the styles as a one-off and call it `LoadingContainer` or something